README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+**Docker Swarm standalone**: This project. An API proxy system in which a single Docker host fronts several Docker Engines to make them appear as if they are a single Docker Engine.
+See [Docker Swarm overview](https://docs.docker.com/swarm/overview/).
+
+**[Swarmkit](https://github.com/docker/swarmkit)**: Cluster management and
+orchestration features in Docker Engine 1.12 or later. When you enable this
+plugin, the Docker Engine runs swarm mode. Swarm mode includes the features
```

Comment:
I don't think we should call it a plugin, since plugins in Docker are different things.